src/lib/controls/Input.svelte

```diff
@@ -33,6 +37,7 @@
 		}
 
 		if (handleValidity(newValue)) value = newValue;
+		dispatch('inputevent');
```

Comment:
You don't have to change anything, I'm just curious why `inputevent` instead of `input`? Does that cause a problem I should watch out for?